test/test_jit.py

```diff
@@ -452,7 +452,7 @@ def checkScript(self,
     def checkTrace(self, func, reference_tensors, input_tensors=None,
                    optimize=True, drop=None, allow_unused=False, verbose=False,
                    inputs_require_grads=True, check_tolerance=1e-5, export_import=True,
-                   _force_outplace=False):
+                   flatten_inputs=True, _force_outplace=False):
```

Comment:
I think we should always flatten inputs, right? Otherwise the call to grad() won't work. Having `flatten_inputs` be a flag makes the code below more complicated, so if it's possible to remove, that would be good.